js/tic_tac_toe.js

```diff
@@ -1,2 +1,52 @@
-// Your code here
-// Use the jQuery
+ $(document).ready(function(){
+ var $box_cell = $('.box_cell');
+
+  var player_1 = function(turn, $box_cell){
+   var turn = 1;
+   turn += 2;
+  $box_cell.on("click", function(){
+       $(this).css('background-color', 'black');
+       });
+  var removeColor = function() {
+    this.remove('background-color');
+  }
+
+ var player_2 = function(){
+  var turn = 2;
+   turn += 2;
+  var $box_cell = $('.box_cell');
+   $box_cell.on("click", function(){
+        $(this).css('background-color', 'red');
+      });
+var removeColor = function() {
+    this.remove('background-color');
+  }
+
+    return { player_1: player_1, player_2: player_2 };
+
+});
+
+var playerTurn = (function(){
+  if (turn === odd_num);
+    return player_1;
+  else (turn === even_num);
+    return player_2;
+})();
+
+//to determine a win,
```

Comment:
I like what you have here to talk about winning states and your logic. That's more important than just the code itself and I think you're on the right path.